client/ContributionsListView.js

```diff
@@ -95,18 +111,26 @@ class MainPageTB extends Component {
                             className={classes.button}>Add Collection </Button>
                         <List className={classes.contributionList}>
                             {contrib.filter(e => e.type).map((e) => {
+                                let pendingApproval = e.approval === "pending";
+                                let published = this.state.publishedList && this.state.publishedList[e.ref.id] === 'true';
```

Comment:
`=== true` is redundant

Reply:
this is a string comparison - the value of this field may also be 'false' if a collection was once published and an admin decided to unpublish it